src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { color, name } = props;
+
+  return (
+    <div className="button" style={{ backgroundColor: color }}>
```

Comment:
Your styling should be in CSS file, but you might wanna use a ternary operator later in the milestone.